App/Listeners/GenerateSitemap.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Listeners;
+
+use TightenCo\Jigsaw\Jigsaw;
+use samdark\sitemap\Sitemap;
+
+class GenerateSitemap
+{
+    public function handle(Jigsaw $jigsaw)
+    {
+        $baseUrl = $jigsaw->getConfig('baseUrl');
+        $sitemap = new Sitemap($jigsaw->getDestinationPath() . '/sitemap.xml');
+
+        collect($jigsaw->getOutputPaths())
+            ->map(function ($path) {
+                if ($path) {
+                    return str_start($path, '/');
+                }
+                return $path;
```

Comment:
Is this necessary?

Reply:
The `return $path` line or the whole map function? The former isn't really necessary. It should just be returning an empty string. Void would probably be fine, so should I remove it?

Reply:
The return path line :) Since it'll just return void without it, then I don't think we need an explicit return; thanks!

Reply:
Sorry, somehow I 'm just now seeing this. I must've missed the notification. It should be good to go now!